tests/e2e/adu/service/create_and_import_manifest.ps1

```diff
@@ -0,0 +1,158 @@
+$accountCredential = [System.Management.Automation.PSCredential]::new($AzureAdClientId, $accountSecret)
+Connect-AzAccount -ServicePrincipal -Credential $accountCredential -Tenant $AzureAdTenantId -ErrorAction Stop
+$subscription = Get-AzSubscription -SubscriptionId $AzureSubscriptionId -ErrorAction Stop
+Set-AzContext $subscription -ErrorAction Stop | Out-Null
+$account = Get-AzStorageAccount -ResourceGroupName $AzureResourceGroupName -Name $AzureStorageAccountName -ErrorAction Stop
+$container = Get-AzStorageContainer -Name $AzureBlobContainerName -Context $account.Context -ErrorAction Stop
+
+# Create new update for v1.1
+$updateId = New-AduUpdateId -Provider ADU-E2E-Tests -Name Linux-E2E-Update -Version 1.1
+$compat = New-AduUpdateCompatibility -Properties @{ deviceManufacturer = 'PC'; deviceModel = 'Linux-E2E' }
+$installStep = New-AduInstallationStep -Handler 'microsoft/swupdate:1'-HandlerProperties @{ installedCriteria = '1.1' } -Files '..\e2e_build\azure_iot_e2e_adu_tests_v1.1'
+$updateInput = New-AduImportUpdateInput -UpdateId $updateId `
+                                         -Compatibility $compat `
+                                         -InstallationSteps $installStep `
+                                         -BlobContainer $container `
+                                         -ErrorAction Stop `
+                                         -Verbose:$VerbosePreference
+
+$secureStringAuth = ConvertTo-SecureString $AzureAdApplicationSecret -AsPlainText -Force
+
+$AuthorizationToken = Get-MsalToken `
+            -ClientId $AzureAdClientId `
+            -TenantId $AzureAdTenantId `
+            -Scope 'https://api.adu.microsoft.com/.default' `
+            -Authority "https://login.microsoftonline.com/$AzureAdTenantId/v2.0" `
+            -ClientSecret $secureStringAuth `
+            -ErrorAction Stop
+
+# Import update into blob storage for deployment
+$operationId = Start-AduImportUpdate -AccountEndpoint $AccountEndpoint `
+                                     -InstanceId $InstanceId `
+                                     -AuthorizationToken $AuthorizationToken.AccessToken `
+                                     -ImportUpdateInput $updateInput `
+                                     -Verbose:$VerbosePreference
+
+Wait-AduUpdateOperation -AccountEndpoint $AccountEndpoint `
+                        -InstanceId $InstanceId `
+                        -AuthorizationToken $AuthorizationToken.AccessToken `
+                        -OperationId $operationId `
+                        -Timeout (New-TimeSpan -Minutes 15) `
+                        -Verbose:$VerbosePreference
+
+
+# Deploy the update to the device group
+$CurrentTime = [DateTime]::UtcNow.ToString('u')
+$DeploymentId = [guid]::NewGuid().toString()
+
+$deploymentUri = "https://$AccountEndpoint/deviceUpdate/$InstanceId/management/groups/$GroupID/deployments/$DeploymentId/?api-version=2021-06-01-preview"
```

Comment:
Will this have to change after they GA? :(

Reply:
They might but for now we have to use the preview. Once it's updated we can update it here.